TwoFAS/TwoFAS/Root/Modules/Settings/AppleWatch/View/AppleWatchViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  This file is part of the 2FAS iOS app (https://github.com/twofas/2fas-ios)
+//  Copyright © 2024 Two Factor Authentication Service, Inc.
+//  Contributed by Grzegorz Machnio. All rights reserved.
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program. If not, see <https://www.gnu.org/licenses/>
+//
+
+import UIKit
+import SwiftUI
+
+final class AppleWatchViewController: UIViewController {
+    var presenter: AppleWatchPresenter!
+
+    override func viewDidLoad() {
+        title = T.Settings.appleWatch
+
+        let installationStepAction: (_ stepNumber: Int) -> Void = { [weak self] stepNumber in
+            self?.presenter.handleInstallationStep(number: stepNumber)
+        }
+        let appleWatchView = AppleWatchView(
+            appleWatchInstallationSteps: [
+                .init(descirption: T.AppleWatch.installationFirstStep,
+                      actionTitle: T.AppleWatch.installationFirstStepLink),
+                .init(descirption: T.AppleWatch.installationSecondStep,
+                      actionTitle: T.AppleWatch.installationSecondStepLink)
+            ],
+            installationStepActionCallback: installationStepAction
```

Comment:
This isn't necessary. We should pass AppleWatchPresenter directly into AppleWatchView and call `onInstallationStep(....` from there.